projects/packages/sync/src/class-functions.php

```diff
@@ -628,4 +628,25 @@ public static function json_wrap( &$any, $seen_nodes = array() ) {
 		return $any;
 
 	}
+
+	/**
+	 * Return the list of installed themes
+	 *
+	 * @return array
+	 */
+	public static function get_themes() {
+		$current_stylesheet = get_stylesheet();
+		$installed_themes   = wp_get_themes();
+		$synced_headers     = array( 'Name', 'ThemeURI', 'Description', 'Author', 'Version', 'Template', 'Status', 'TextDomain', 'RequiresWP', 'RequiresPHP' );
```

Comment:
Is this enough? Or too much? Description can be a very long string...

Should we add a filter to this list? Or a filter to the function output at the end? In case a plugin needs something specific?

Reply:
@bisko any thought here?